Smartpad-macOS/Smartpad-macOS/Logic/ConnectionManager.swift

```diff
@@ -50,13 +56,14 @@ class ConnectionManager:NSObject, MCSessionDelegate, MCNearbyServiceBrowserDeleg
         mcBrowser.maximumNumberOfPeers = 1
         listVC.presentAsSheet(mcBrowser)
     }
-    
 
     /**
      Method beings browsing for other peers
      */
     func searchForDevices(){
-        guard let p2pSession = p2pSession else {return}
+        /* Restart the session to update our display name */
+        restartP2PSession()
```

Comment:
I have to restart the p2p session each time because the device name cannot be updated while the session is open